package.json

```diff
@@ -58,6 +58,7 @@
     "fake-indexeddb": "1.0.3",
     "file-loader": "^0.8.5",
     "gh-pages": "^0.11.0",
+    "history": "^1.17.0",
```

Comment:
A peer dependency which was handled automatically just fine with npm 2 but which breaks with npm 3; declaring it here makes it work in both versions :)